### PR TITLE
Fortran, this should fix the speed issue

### PR DESCRIPTION
**Pull request by francescoalemanno**
I run the CI service 2 times using the ubuntu image and now it seems to be working fine...

```
            +fortran | Started:            
1037
            +fortran | Started:            18-10-2022 21:25:04 
1038
            +fortran | Ended:              18-10-2022 21:25:04 
1039
            +fortran | Language:           Fortran 90 (gfortran) 
1040
            +fortran | Version:            11.2.0 
1041
            +fortran | Executed Command:   ./leibniz
1042
            +fortran | Executed Command:   ./leibniz 
1043
            +fortran | Accuracy:           
1044
            +fortran | Accuracy:           0.5555555555555556 
1045
            +fortran | Calculated Pi:      3.1415926437128880 
1046
            +fortran | Total iterations:   3 
1047
            +fortran | Average time taken: 
1048
            +fortran | Average time taken: 69.107386ms 

```
vs cpp
```

                +cpp | Started:            
316
                +cpp | Started:            18-10-2022 21:24:01 
317
                +cpp | Ended:              18-10-2022 21:24:01 
318
                +cpp | Language:           C++ (g++) 
319
                +cpp | Version:            11.2.1 
320
                +cpp | Executed Command:   ./leibniz 
321
                +cpp | Accuracy:           0.7222222222222222 
322
                +cpp | Calculated Pi:      3.1415926635893259 
323
                +cpp | Total iterations:   3 
324
                +cpp | Average time taken: 135.450249ms 
```

### CODE REVIEW

File: Earthfile [hunk 1]
```diff
@@ -133,11 +133,13 @@ elixir:
...
-  RUN apk add --no-cache gfortran build-base
-
+  FROM ubuntu:latest
+  RUN apt-get update
+  RUN apt-get install -y gfortran
+  COPY ./src/rounds.txt ./
+  COPY +build/scbench ./
   COPY ./src/leibniz.f90 ./
-  RUN --no-cache gfortran -Ofast -flto leibniz.f90 -o leibniz
+  RUN --no-cache gfortran -Ofast -march=native -mtune=native -flto -ffast-math leibniz.f90 -o leibniz
```
Comment (giordano) [inline Earthfile:142]: Same comment as https://github.com/niklas-heer/speed-comparison/pull/61#issuecomment-1283079377 :laughing:

**Comment by Moelf** [comment]
>+fortran | Accuracy:           0.5555555555555556 

eh...

**Comment by francescoalemanno** [comment]
actually:

```
:::   pi-3.1415926635893259  for CPP
-9.999532757376528e-9

:::   pi-3.1415926437128880 for FORTRAN
9.876905071593e-9
```

File: Earthfile (hunk 1 above)
Comment (francescoalemanno) [reply Earthfile:142]: :) see comment below.

**Review by niklas-heer — APPROVED** [review]

**Comment by niklas-heer** [comment]
@francescoalemanno I had to update it due to #66

**Review by niklas-heer — APPROVED** [review]

**Comment by niklas-heer** [comment]
Thank you for your contribution @francescoalemanno 👍

**Pull request merged**